### ISSUE TITLE
Implement continuous flashing mode

### ISSUE THREAD

**Issue by jhermsmeier** [resin-io/etcher#1805]
This would be a mode with no target selection, once "flash" is hit all available targets are written to.

Depends on: #1806

**Comment by lurch** [comment]
Duplicate of #951 ?

**Comment by jhermsmeier** [comment]
Duplicate of #951
